nx-X11/lib/X11/XlibInt.c

```diff
@@ -2751,6 +2751,25 @@ void _XEatData(
 #undef SCRATCHSIZE
 }
 
+/*
+ * Read and discard "n" 32-bit words of data
+ * Matches the units of the length field in X protocol replies, and provides
+ * a single implementation of overflow checking to avoid having to replicate
+ * those checks in every caller.
+ */
+#ifndef HAVE__XEATDATAWORDS
```

Comment:
I think he invented it deliberately, so that we can just disable the function here should that need arise.

Not quite sure why we'd want to do this, if it's really missing and messing up Xrender, though...